src/html_tag.cpp

```diff
@@ -2751,10 +2751,15 @@ litehtml::element::ptr litehtml::html_tag::get_child( int idx ) const
 
 size_t litehtml::html_tag::get_index() const
 {
+	return get_index( this );
+}
+
+size_t litehtml::html_tag::get_index( const litehtml::element * element ) const
+{

```

Comment:
I don't understand the goal of that argument. The code is wrong at different level : 
- It doesn't use anything else than parent from 'this'
- Parent of passed argument might be different than 'this' parent and is not checked
- I don't see it use here, so I can't infer usage, but seems strange to do el->get_index( other ) while other->get_index() works 

What was the goal of that change?